type.proto

```diff
@@ -197,6 +197,18 @@ message DisruptionProperty {
     required string value                           = 3;
 }
 
+message TimeSlot {
+    // Times in local for a timeslot
+    required uint32 begin                           = 1;
+    required uint32 end                             = 2;
```

Comment:
out of curiosity, how is the time being represented : 
* as a 6 digits number (ie "08:30" -> int(830) )
* as a number of second from zero ? (ie "08:30" -> int(520) )

Reply:
begin(end): normally in the protobuf, it is the number of seconds from midnight

Reply:
it's probably worth documenting it :)